README.md

```diff
@@ -116,18 +116,120 @@ One of the big concerns with IPFS, and even cloud data storage in general is enc
+    exit 1
+fi
+if [[ "$GOVERSION" -lt 12 ]]; then
+    echo "[WARN] detected golang version is less than 1.12 and may produce errors"
+fi
+# create working directory
+mkdir "$WORKDIR"
+cd "$WORKDIR"
+# download temporal
+git clone https://github.com/RTradeLtd/Temporal.git
+cd Temporal
+# initialize submodules, and download all dependencies
+make setup
+# make cli binary
+make install
+```
+
+**Downloading Pre-Built Binaries:**
+
+To download pre-built binaries, currently available for Linux and Mac OSX platforms, head on over to our [github releases page](https://github.com/RTradeLtd/Temporal/releases/latest).
+
+Download a binary for your appropriate platform, additionally you can download sha256 checksums of the released binaries for post-download integrity verification. 
+
+You'll then want to copy the pre-built binary over to anywhere in your `PATH` environment variable.
+
+Example for downloading and installing the Linux version:
+
+```bash
+# download the binary
+wget https://github.com/RTradeLtd/Temporal/releases/download/v2.2.7/temporal-v2.2.7-linux-amd64
+# download the binary checksum
+wget https://github.com/RTradeLtd/Temporal/releases/download/v2.2.7/temporal-v2.2.7-linux-amd64.sha256
+# store downloaded checksum output
+CK_HASH=$(cat *.sha256  | awk '{print $1}')
+# calculate sha256 checksum of downloaded binary
+DL_HASH=$(sha256sum temporal-v2.2.7-linux-amd64 | awk '{print $1}')
+# compare checksums, and copy file to `PATH` if ok
+# if this doesn't show ok then your binary is corrupted or has been tampered with
+if [[ "$CK_HASH" == "$DL_HASH" ]]; then  echo "ok" && sudo cp temporal-v2.2.7-linux-amd64 /usr/local/bin; fi
+```
+
+### Configuration Initialization
+
+After downloading Temporal, regardless of your setup process you will need a configuration file. To generate a config file at `/home/doggo/config.json` run the following command:
+```
+temporal -config /home/doggo/config.json init
+```
+**Alternatively you can set the environment variable `CONFIG_DAG` and `temporal init`, along with *all other commands* will read from this location. It is recommended that you do this as it makes using the cli a lot easier**
+
+It is **extremely** important you keep this in a directory that is only accessible to the users required to run the servivce as it contains usernames and passwords to key pieces of Temporal's infrastructure.
+
+For an example bare-minium configuration file, check out [testenv/config.json](https://github.com/RTradeLtd/testenv/blob/master/config.json)
+
+Note that if you did the install from source method, you will already have a config file in your home directory called `temporal-config.json`.
+
+### Manual Setup
+
+This exact process will vary a bit depending on the environment you are installing Temporal in. At the very least you are required to use Postgres, and RabbitMQ. The operating systems you install those, and the supplementary services on is entirely up to you, but we recommend using Ubuntu 18.04LTS.
```

Comment:
Would be ideal to add links to guides on how to set these up and anything beyond the vanilla install that you need to use with temporal. 

Also to note what details about these databases you need to put into the config file (and where to put them)

Reply:
These are actually located in the confluence page that details installing Temporal in production. I have made a note in the readme that it is covered there.